CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+### Fixed
+- Exchange backups would fail attemptting to use delta tokens even if the user was over quota
```

Comment:
Welcome to the club!
```suggestion
- Exchange backups would fail attempting to use delta tokens even if the user was over quota
```

Reply:
oh, I think this one is just me always being terrible at spelling. I don't think my keyboard has turned against me... yet...

Reply:
That's what I said, too.